lib/swagger/index.js

```diff
@@ -17,7 +17,8 @@ swagger._getSwaggerBase = () => {
   let basePath, host, protocol
   if (jsonApi._apiConfig.urlPrefixAlias) {
     const urlObj = url.parse(jsonApi._apiConfig.urlPrefixAlias)
-    basePath = urlObj.pathname.replace(/\/$/, '')
```

Comment:
Had to use something slightly different as javascript does not have a look behind in it's regex capabilities. The alternative works to the same effect: (?!^\/)\/$